app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemInfoTabs.jsx

```diff
@@ -243,11 +243,20 @@ export class LogItemInfoTabs extends Component {
     ) : null;
   };
 
+  getActiveTabId = () => {
+    const { activeTabId } = this.state;
+    const tabs = this.makeTabs();
```

Comment:
Please, call the method `makeTabs()` only once in the render method to avoid unnecessary calculations.